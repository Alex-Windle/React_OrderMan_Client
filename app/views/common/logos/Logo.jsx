/**
 * GENERAL NOTES
 * @author TalkRise <admin@talkrise.com>
 */


// Module imports
import React, { Component } from 'react';
import logoImage from '../../../assets/logo.png';


/**
 * Logo represents the OrderMan project logo
 *
 * @description Renders a responsive image of the logo
 *
 * @extends Component
 */
export default class Logo extends Component {
  render() {
    return (
      <img
        src={logoImage}
        alt="TalkRise OrderMan"
        className="img"
        width="150"
      />
    );
  }
}
