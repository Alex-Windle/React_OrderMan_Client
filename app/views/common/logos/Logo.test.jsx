/**
 * GENERAL NOTES
 * @author TalkRise <admin@talkrise.com>
 */


// Module imports
import React from 'react';
import renderer from 'react-test-renderer';


// Component imports
import Logo from './Logo';


describe('Logo component', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(
      <Logo />,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
