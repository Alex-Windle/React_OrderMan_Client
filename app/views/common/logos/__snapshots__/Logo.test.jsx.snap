// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component matches snapshot 1`] = `
<img
  alt="TalkRise OrderMan"
  className="img"
  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+P+/HgAFhAJ/wlseKgAAAABJRU5ErkJggg=="
  width="150"
/>
`;
